src/components/NavTabs.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import '../pages/HomePage.css';
+
+const NavTabs = () => {
```

Comment:
@ajiles91 @MonicaDJohnson @mikeramz86 I put the navigation tabs in a new component and included it on every page we have so far. Do you approve of this change? I didn't change any code.

Here's a sandbox showing the changes.
https://codesandbox.io/s/github/the-collab-lab/tcl-3-smart-shopping-list/tree/aj-np-create-new-list-token/?fontsize=14&hidenavigation=1&theme=dark